frontend/src/restapi/tournament-registrations/TournamentRegistrationsRestAPI.ts

```diff
@@ -55,9 +56,15 @@ export const TournamentRegistrationsRestApi = (
         .then((res) => res.data);
     },
     async createTournament(body: { tournamentId: string }): Promise<void> {
+      const cookies = new Cookies();
       await axios.post(
```

Comment:
Fajnie byloby to zrefaktorowac, np. tutaj podadwac instancje axiosa, ktora zawsze uzywa tego.